templates/page.values.php

```diff
@@ -49,6 +49,6 @@
 			</li>
 		</ul>
 
-		<p class="details-link"><a href="<?php p($theme->getBaseUrl()); ?>" target="_blank" rel="noreferrer noopener"><?php p($l->t('Learn more about %s', $theme->getName())); ?></a></p>
+		<p class="details-link"><a href="https://nextcloud.com" target="_blank" rel="noreferrer noopener"><?php p($l->t('Learn more about Nextcloud')); ?></a></p>
```

Comment:
Please use https://github.com/nextcloud/server/blob/3860dad9eaac937288d125c75e7f58cfbe44848a/lib/public/Defaults.php#L130 for the product name which would allow to specify the software. For the product url we currently don't have anything to replace, but we might need to have that adaptable for service provider scenarios.

Reply:
the first part is done,  the second for specifying the url is not

Reply:
> For the product url we currently don't have anything to replace, but we might need to have that adaptable for service provider scenarios.

Any idea how to do that?